src/util/defined.js

```diff
@@ -0,0 +1,11 @@
+// Checks that passes properties are 'defined', meaning that calling them with d, i returns non null values
+export default function(d, i) {
+    for (var c = 0, j = arguments.length; c < j; c++) {
+        if (arguments[c] !== d && arguments[c] !== i) {
```

Comment:
Ah ... I can see why your code in `boxBase` passed `d` and `i`!

What this function should do is take a list of functions, returning a new function which you invoke like so:

```
function upper(d,i) { ... };
function lower(d,i) { ... };
function center(d,i) { ... };

defined(upper, lower, center)(d, i);
```

Does that make sense?

Considering the usage above, I'd move this to the `fc.util.fn` namespace.